### PR TITLE
fix: popover overflow and truncation for breadcrumb

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#681

## Description
This PR removes overflow from popover so that the corners do not look weird. It also applies proper fiori 3 styling using dropdown styling for the truncation.
### Before:
Screen Shot 2020-03-20 at 10 36 58 AM

### After:
Screen Shot 2020-03-20 at 10 35 20 AM

### CODE REVIEW

File: src/list.scss [hunk 1]
```diff
@@ -357,11 +365,11 @@ $block: #{$fd-namespace}-list;
...
-        border-bottom-right-radius: $fd-list-last-item-bottom-border-radius;
-      }
-    }
+    // > * {
+    //   &:last-child {
+    //     border-bottom-left-radius: $fd-list-last-item-bottom-border-radius;
+    //     border-bottom-right-radius: $fd-list-last-item-bottom-border-radius;
+    //   }
+    // }
   }
```
Comment (InnaAtanasova) [inline src/list.scss]: Please remove this if we don't need it

**Comment by stefanoScalzo** [comment]
> 1. The body of the popover should have rounded corners.
>    Here is how it looks now:
> 
> Screen Shot 2020-03-20 at 12 08 11 PM
> 
> Here is how it should look like (as per specs)
> Screen Shot 2020-03-20 at 12 04 15 PM
> 
> 1. The focus outline is not correct. Please check `Focus Visual Concept ` in the specs for more info.
>    Here is how it looks in your PR:
> 
> Screen Shot 2020-03-20 at 12 08 11 PM
> 
> How it looks in Select and MultiInput. All these components follow the List specs:
> Screen Shot 2020-03-20 at 12 09 00 PM
> 
> 1. Please maintain the right section of the PR in GitHub. I did it for you now but I can't do it for every PR
> 
> Screen Shot 2020-03-20 at 12 10 00 PM

Even though the image on the breadcrumb page has rounded corners it says: Breadcrumbs are responsive. If there is insufficient horizontal space, the links in the breadcrumb trail collapse into a dropdown menu (sap.m.Select); The select component does not have rounded top corners.

More detail on what is the issue with the focus please, it has the proper styling, offset and width

File: src/list.scss (hunk 1 above)
Comment (stefanoScalzo) [reply src/list.scss]: fixed

**Comment by InnaAtanasova** [comment]
> > 1. The body of the popover should have rounded corners.
> >    Here is how it looks now:
> > 
> > Screen Shot 2020-03-20 at 12 08 11 PM
> > Here is how it should look like (as per specs)
> > Screen Shot 2020-03-20 at 12 04 15 PM
> > 
> > 1. The focus outline is not correct. Please check `Focus Visual Concept ` in the specs for more info.
> >    Here is how it looks in your PR:
> > 
> > Screen Shot 2020-03-20 at 12 08 11 PM
> > How it looks in Select and MultiInput. All these components follow the List specs:
> > Screen Shot 2020-03-20 at 12 09 00 PM
> > 
> > 1. Please maintain the right section of the PR in GitHub. I did it for you now but I can't do it for every PR
> > 
> > Screen Shot 2020-03-20 at 12 10 00 PM
> 
> Even though the image on the breadcrumb page has rounded corners it says: Breadcrumbs are responsive. If there is insufficient horizontal space, the links in the breadcrumb trail collapse into a dropdown menu (sap.m.Select); The select component does not have rounded top corners.
> 
> More detail on what is the issue with the focus please, it has the proper styling, offset and width

@stefanoScalzo  If there is discrepancy in the specs the approach is to reach the designers and ask them for confirmation which one - the visuals or the text is correct.

For the offset - please check the offset distance. From what I see it's bigger than it should be

**Pull request closed without merging**